migrations/2024-04-23-132120_add-extended-card-info-to-business-profile/up.sql

```diff
@@ -0,0 +1,3 @@
+-- Your SQL goes here
+
+ALTER TABLE business_profile ADD COLUMN IF NOT EXISTS is_extended_card_info_enabled BOOLEAN DEFAULT FALSE;
```

Comment:
```suggestion
ALTER TABLE business_profile ADD COLUMN IF NOT EXISTS store_extended_card_bin BOOLEAN DEFAULT FALSE;
```

Reply:
This PR is not about card bin. That is being picked up by @Aprabhat19 . This is a different usecase